### PR TITLE
Add paas-sig repo to be able to install Origin

### PR DESCRIPTION
**Pull request by jhadvig**
@stevekuznetsov PTAL

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
This looks good -- can you also add the `openshift-future` repo under the same domain?

**Comment by jhadvig** [comment]
Sir, yes sir :+1:

**Comment by jhadvig** [comment]
not sure what `openshift-future` repo you mean ?

**Comment by stevekuznetsov** [comment]
There should be a repository named `openshift-future` from the CentOS PaaS SIG under the same domain as the one you're touching with this PR.

File: oct/ansible/oct/roles/dependencies/tasks/register_repositories.yml [hunk 1]
```diff
@@ -50,14 +50,17 @@
...
+    description: '{{ item.description }}'
+    baseurl: '{{ item.url }}'
     gpgcheck: yes
     sslverify: no
     sslclientcert: /var/lib/yum/client-cert.pem
     sslclientkey: /var/lib/yum/client-key.pem
-  when: ansible_distribution == 'CentOS'
\ No newline at end of file
+  with_items:
+    - { name: 'centos-paas-sig-openshift-origin-rpms', description: 'CentOS PaaS SIG Origin Repository', url: 'http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/' }
```
Comment (stevekuznetsov) [inline oct/ansible/oct/roles/dependencies/tasks/register_repositories.yml]: Can you format this using YAML:

```yml
with_items:
  - name: 'centos-paas-sig-openshift-origin-rpms'
    description: 'CentOS PaaS SIG Origin Repository'
    baseurl: 'http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/'
# etc ...
```

**Comment by jhadvig** [comment]
@stevekuznetsov updated PTAL

**Pull request merged**